src/MediaWiki/Jobs/ChangePropagationDispatchJob.php

```
@@ -276,34 +294,31 @@ private function pushChangePropagationDispatchJob( int $num, $chunk ): void {
 	}
 
 	private function dispatchFromData( WikiPage $subject, $data ): bool {
-		$applicationFactory = ApplicationFactory::getInstance();
-		$cache = $applicationFactory->getCache();
-
 		$property = Property::newFromUserLabel(
 			$this->getTitle()->getText()
 		);
 
-		$semanticData = $applicationFactory->getStore()->getSemanticData(
+		$semanticData = $this->store->getSemanticData(
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$semanticData'. (undefined)

(UnusedLocalVariable)